src/runtime/breakpoints.ts

```diff
@@ -0,0 +1,3 @@
+const style = getComputedStyle(document.documentElement);
+
+export const BREAKPOINT_LARGE = style.getPropertyValue("--docs-breakpoint-lg");
```

Comment:
Samma sak som med scss-filen, ska vi lägga in samtliga direkt för kompletthet?

Reply:
La till resterande breakpoints.